Description: re-format the help option
 Attempt to re-format the help option of the 3dldf
 interpreter wrt to GNU traditions.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-05-22

--- a/src/main.web
+++ b/src/main.web
@@ -450,53 +450,44 @@
 
         else if (option_index == HELP_INDEX)
           {
-            cerr_mutex.lock(); 
-            cerr << "GNU 3DLDF Version " << VERSION_3DLDF << ". "
-                 << COPYRIGHT_3DLDF << endl 
-                 << endl
-                 << "Valid options for GNU 3DLDF are:" << endl
-                 << endl
-                 << "--bison-trace:           Prints information generated by Bison "
-                 << endl << "                         "
-                 << "for tracing which parser rules are reduced "
-                 << endl << "                         "
-                 << "and the state of the Bison stack." << endl
-                 << endl
-                 << "--help:                  Prints this message and exits "
-                 << "with return value 0." << endl
-                 << endl
-                 << "--multithread-input:     Enables the use of multiple "
-                 << "threads for input."
-                 << endl
-                 << endl
-                 << "--multithread-output:    Enables the use of multiple "
-                 << "threads for output."
-                 << endl
-                 << endl
-                 << "--multithread-include:   Enables the use of multiple "
-                 << "threads for input" << endl 
-                 << "                         files included in other input files."
-                 << endl
-                 << endl
-                 << "--quiet:                 Suppresses some output to standard "
-                 << "output"
-                 << endl 
-                 << "                         and standard error when `3dldf' is run." 
-                 << endl << endl
-                 << "--silent:                Suppresses almost all output "
-                 << "to standard output" << endl
-                 << "                         and standard error when "
-                 << "`3dldf' is run."
-                 << endl << endl
-                 << "--verbose:               Causes status information to be "
-                 << "printed to standard output" << endl 
-                 << "                         when `3dldf' is run."
-                 << endl << endl 
-                 << "--version:               Prints the version number of "
-                 << "GNU 3DLDF" << endl 
-                 << "                         to standard output and exits with "
-                 << "return value 0."
-                 << endl << endl; 
+            cerr_mutex.lock();
+            cerr
+						     << "Usage: 3ldf [OPTION]... 3DLDFNAME[.ldf]"
+								   << endl << endl
+						     << "This is GNU 3DLDF Version " << VERSION_3DLDF << ". "
+								   << endl
+								 << COPYRIGHT_3DLDF << "."
+								   << endl << endl
+                 << "Valid options for GNU 3DLDF are:"
+								   << endl << endl
+                 << "  --bison-trace          Prints information generated by Bison "
+								   << "for tracing which parser rules are reduced "
+                   << "and the state of the Bison stack"
+								   << endl << endl
+                 << "  --multithread-input    Enables the use of multiple threads for input"
+                   << endl << endl
+                 << "  --multithread-output   Enables the use of multiple threads for output"
+                   << endl << endl
+                 << "  --multithread-include  Enables the use of multiple threads for input"
+                   << "files included in other input files"
+                   << endl << endl
+                 << "  --quiet                Suppresses some output to standard output"
+                   << "and standard error when `3dldf' is running"
+                   << endl << endl
+                 << "  --silent               Suppresses almost all output to standard output"
+                   << "and standard error when `3dldf' is running"
+                   << endl << endl
+                 << "  --verbose              Causes status information to be "
+                   << "printed to standard output when `3dldf' is running"
+                   << endl << endl
+                 << "  --help                 Prints this message and exits with return value 0"
+                   << endl << endl
+                 << "  --version              Prints the version number of GNU 3DLDF "
+                   << "to standard output and exits with return value 0"
+                   << endl << endl
+                 << "Report bugs to <" << PACKAGE_BUGREPORT << ">."
+								   << endl
+                 ;
 
 
 #if DEBUG_COMPILE
